python/grass/ctypes/loader.py

```diff
@@ -238,6 +238,12 @@ def __getattr__(self, name):
 class WindowsLibraryLoader(LibraryLoader):
     name_formats = ["%s.dll", "lib%s.dll"]
 
+    def __init__(self):
```

Comment:
@nilason isn't such a change/addition already included in one of yours PRs?

Reply:
Yes, in #1651, but that one is based upon this change, so this is good.

Reply:
so, we need it just in one place, right? :-)

Reply:
Certainly, but that PR should only be merged after this one is merged and it had a rebase.